app/src/main/java/com/orange/ouds/app/ui/components/ComponentsScreen.kt

```diff
@@ -52,6 +55,7 @@ private fun ComponentsScreen(components: List<Component>, onComponentClick: (Lon
                     onClick = { onComponentClick(component.id) }
                 )
             }
+            Spacer(modifier = Modifier.height(OudsNavigationBarHeight))
```

Comment:
I wonder if there is a more conventional way of doing this with the window insets. I did not take time to look at this but I can investigate that point (I need to read the documentation again because I forgot everything 😅).